src/layer/marker/DivIcon.js

```diff
@@ -1,3 +1,6 @@
+import {Icon} from './Icon';
+import {toPoint as point} from '../../geometry/Point';
```

Comment:
And here, `as` is not needed — `toPoint` is descriptive and doesn't conflict with `point` variables.